ui/Screen/Onboarding/Success.svelte

```diff
@@ -40,16 +40,13 @@
   <div class="content">
     <h1 style="text-align: center; margin-bottom: 1.5rem;">All set!</h1>
 
-    <Urn
-      urn={id}
-      showCopyOnlyOnHover
-      notificationText={`Radicle ID copied to your clipboard.`} />
+    <ShareableIdentifier urn={id} />
 
     <p
       style="text-align: center; width: 23.13rem; margin: 1.75rem 0 1.75rem 0;
       color: var(--color-foreground-level-6);">
-      This is your Radicle ID — it's unique to you!<br /> Click to copy it and share
-      it with others so that they can find you.
+      This is your handle — it's unique to you!<br /> Click to copy it and share
```

Comment:
This has been changed in some other context, but quite confusing to refer to the id as a handle. Where in other products it's used for the "nickname". /cc @cloudhead

Reply:
Yeah, we've discussed this at length -- we don't have an exact equivalent to the "handle" on radicle, because our only unique identifier is a long random string. So we use the term handle as "unique id", in the same way `cloudhead` is my unique id/handle on social media.

Now the thing is, this choice was made in the context of an id that looked like a handle, ie. `@abcedf123`. Now that we're using a sort of hybrid of that + a URN on copy, the term "handle" doesn't work as well. I've been thinking about this and I don't like where we're at with all this. I think we'll have to revisit a lot of it after the B1.